ocs_ci/ocs/uninstall.py

```diff
@@ -37,23 +37,112 @@ def remove_ocp_registry_from_ocs(platform):
 
     """
     image_registry_obj = ocp.OCP(
-        kind=constants.CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+        kind=constants.IMAGE_REGISTRY_CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
     )
-    param_cmd = None
+    params_list = list()
     if platform.lower() == constants.AWS_PLATFORM:
-        param_cmd = '[{"op": "remove", "path": "/spec/storage"}]'
+        params_list.append('[{"op": "remove", "path": "/spec/storage"}]')
+        params_list.append('[{"op": "remove", "path": "/status/storage"}]')
 
     elif platform.lower() == constants.VSPHERE_PLATFORM:
-        param_cmd = '[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]'
+        params_list.append('[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]')
+        params_list.append('[{"op": "replace", "path": "/status/storage", "value": {"emptyDir": "{}"}}]')
 
-    if param_cmd:
-        image_registry_obj.patch(
-            resource_name=constants.IMAGE_REGISTRY_RESOURCE_NAME, params=param_cmd, format_type='json'
-        )
+    if params_list:
+        for params in params_list:
+            image_registry_obj.patch(params=params, format_type='json')
     else:
         log.info("platform registry not supported")
 
 
+def remove_cluster_logging_operator_from_ocs():
+    """
+    Function removes cluster logging operator from OCS
+
+    """
+    # Deleting the clusterlogging instance
+    clusterlogging_obj = ocp.OCP(
+        kind=constants.CLUSTER_LOGGING, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    )
+    assert clusterlogging_obj.delete(resource_name='instance')
+
+    # Deleting the PVCs
+    pvc_obj = ocp.OCP(
+        kind=constants.PVC, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    )
+    pvc_list = get_all_pvcs(namespace=constants.OPENSHIFT_LOGGING_NAMESPACE)
+    for pvc in range(len(pvc_list) - 1):
+        pvc_obj.delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
+        pvc_obj.wait_for_delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
```

Comment:
can we cycle on the PVCs for waiting for deletion after we initiate deletion for all of them?

Reply:
I think this wait can be removed anyway - delete has a wait param set to wait by default